.fossa.yml

```diff
@@ -0,0 +1,18 @@
+# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
+# Visit https://fossa.com to learn more
+
+version: 2
+cli:
+  server: https://app.fossa.com
+  fetcher: custom
+  project: git@github.com:auth0/auth0-deploy-cli.git
+analyze:
+  modules:
+  - name: .
+    type: npm
+    target: .
+    path: .
+  - name: .
```

Comment:
@NotMyself Do you know why the elements under `modules` are duplicated?

Reply:
I do not.